containers/docker/README.md

```diff
@@ -1,58 +1,37 @@
-
-## Tags
+## Supported database backends
 
-| Tag Name | Description |
+| Backend | Description|
 |-|-|
-| `latest` | Builds off of the latest code.  May occasionally be unstable. |
-| `release` | Points to the latest _stable_ release. |
-| `X.X.X` | Semantic version tags. |
-
-When in doubt, use `release`.
+| **ephemeral** | SQLite3 backed container that runs alone.  All data is lost after a container restart. This is set by default|
+| **postgres** | Postgres backed container that can be pointed to a pre-existing database instance using an environment variable (`DATABASE_URL`).|
+| **mysql** | MySQL or Mariadb backed container that can be pointed to a pre-existing database instance using an environment variable (`DATABASE_URL`).|
 
-`amd64` and `arm64` architectures are both built.  Note [this bug](https://github.com/pglombardo/PasswordPusher/issues/268) in regards to tag availability for the `arm64` architecture.
-
-# Docker Compose
-
-For a quick boot of a database backed application, see the available Docker Compose files:
+## ephemeral
 
-* [pwpush-postgres](https://github.com/pglombardo/PasswordPusher/blob/master/containers/docker/pwpush-postgres/docker-compose.yml)
-* [pwpush-mysql](https://github.com/pglombardo/PasswordPusher/blob/master/containers/docker/pwpush-mysql/docker-compose.yml)
-
-# Docker Containers
-
-## pwpush-ephemeral
-
-This is a single container that runs independently using sqlite3 with no persistent storage (if you recreate the container the data is lost); best if don't care too much about the data and and looking for simplicity in deployment.
+This configuration runs independently using sqlite3 with no persistent storage (if you recreate the container the data is lost); best if don't care too much about the data and and looking for simplicity in deployment.
 
 To run an ephemeral version of Password Pusher that saves no data after a container restart:
-`docker run -p "8000:5100" pglombardo/pwpush-ephemeral:latest`
+`docker run -p "8000:5100" pglombardo/pwpush:latest`
 
 _This example is set to listen on port 8000 for requests e.g. http://0.0.0.0:8000._
 
-Available on Docker hub: [pwpush-ephemeral](https://hub.docker.com/repository/docker/pglombardo/pwpush-ephemeral)
-
-See also this discussion if you want to persist data across container restarts: [pwpush-ephemeral: How to Add Persistence?](https://github.com/pglombardo/PasswordPusher/discussions/448)
+See also this discussion if you want to persist data across container restarts: [pwpush-ephemeral: How to Add Persistence?](https://github.com/pglombardo/PasswordPusher/discussions/448)  
+(Since this link refers to an outdated ephemeral image keep in mind to use current image `pglombardo/pwpush`)
 
-## pwpush-postgres
+## postgres
 
-This container uses a default database URL of:
+To setup the container to use the postgres database backend DATABASE_URL environment variable needs to be configured. The syntax should look like:
```

Comment:
```suggestion
To setup the container to use the PostgreSQL database backend DATABASE_URL environment variable needs to be configured. The syntax should look like:
```

Reply:
Keeping `postgres` was indented since the header is named like this. 
But your proposal looks better so I'll change.

btw: We shouldn't rename the header. I think this will break some links